### PR TITLE
Adding the README_es.md and reference to it in the others files readme

### PR DESCRIPTION
**Pull request by Oussamaosman02**
# What does this PR do?
This PR addede the readme.md file in spanish  version and updated the other readme files to reference it correctly.
I made this because you have the docs in spanish lenguage but no readme was in spanish

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

- @sgugger

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Thanks a lot for your PR. To make sure your new README is kept in sync (at least for the big model tables), could you add it to [this dictionary](https://github.com/huggingface/transformers/blob/83dc49b69b6a6d904d4dd6c90534ea37351af439/utils/check_copies.py#L39) with the standard translations?

Also @osanseviero do you want to proofread this quickly?

**Comment by Oussamaosman02** [comment]
@sguggerI think it's done! :smiley: , If there is something else,tell me

File: README_es.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+🤗 Transformers está respaldado por las tres bibliotecas de deep learning más populares — [Jax](https://jax.readthedocs.io/en/latest/), [PyTorch](https://pytorch.org/) y [TensorFlow](https://www.tensorflow.org/) — con una perfecta integración entre ellos. Es sencillo entrenar sus modelos con uno antes de cargarlos para la inferencia con el otro.
+
+## Demostraciones en línea
+
+Puedes probar la mayoría de nuestros modelos directamente en sus páginas desde el [centro de modelos](https://huggingface.co/models). También ofrecemos [alojamiento de modelos privados, control de versiones y una API de inferencia](https://huggingface.co/pricing) para modelos públicos y privados.
+
+Aquí hay algunos ejemplos:
+
+ En procesamiento del lenguaje natural:
+- [Terminación de palabras enmascaradas con BERT](https://huggingface.co/bert-base-uncased?text=Paris+is+the+%5BMASK%5D+of+France)
```
Comment (osanseviero) [inline README_es.md:80]:
```suggestion
- [Autocompletado de palabras enmascaradas con BERT](https://huggingface.co/bert-base-uncased?text=Paris+is+the+%5BMASK%5D+of+France)
```

File: README_es.md [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+- [Traducción con T5](https://huggingface.co/t5-base?text=My+name+is+Wolfgang+and+I+live+in+Berlin)
+
+En visión de ordenador:
```
Comment (osanseviero) [inline README_es.md:88]:
```suggestion
En visión de computadora:
```

File: README_es.md [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+- [Detección de objetos con DETR](https://huggingface.co/facebook/detr-resnet-50)
+- [Segmentación semántica con SegFormer](https://huggingface.co/nvidia/segformer-b0-finetuned-ade-512-512)
+- [Segmentación panóptica con DETR](https://huggingface.co/facebook/detr-resnet-50-panoptic)
+
+En Audio:
+- [Reconocimiento de voz automático con Wav2Vec2](https://huggingface.co/facebook/wav2vec2-base-960h)
+- [Detección de palabras clave con Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-ks)
+
+En tareas multimodales:
+- [Respuesta visual a preguntas con ViLT](https://huggingface.co/dandelin/vilt-b32-finetuned-vqa)
```
Comment (osanseviero) [inline README_es.md:99]:
```suggestion
En tareas multimodales:
- [Respuesta de pregntas sobre imágenes con ViLT](https://huggingface.co/dandelin/vilt-b32-finetuned-vqa)
```

File: README_es.md [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+
+# Allocate a pipeline for sentiment-analysis
+>>> classifier = pipeline('sentiment-analysis')
+>>> classifier('We are very happy to introduce pipeline to the transformers repository.')
+[{'label': 'POSITIVE', 'score': 0.9996980428695679}]
+```
+
+La segunda línea de código descarga y almacena en caché el modelo previamente entrenado que usa la canalización, mientras que la tercera lo evalúa en el texto dado. Aquí la respuesta es "positiva" con una confianza del 99,97%.
+
+Muchas tareas tienen un `pipeline` preentrenado listo para funcionar, en NLP pero también en visión por ordenador y habla. Por ejemplo, podemos extraer fácilmente los objetos detectados en una imagen:
```
Comment (osanseviero) [inline README_es.md:124]:
```suggestion
Muchas tareas tienen un `pipeline` preentrenado listo para funcionar, en NLP pero también en visión por computadora y tareas de audio. Por ejemplo, podemos extraer fácilmente los objetos detectados en una imagen:
```

File: README_es.md [hunk 5]
```diff
@@ -0,0 +1,443 @@
...
+  'box': {'xmin': 0, 'ymin': 1, 'xmax': 639, 'ymax': 473}},
+ {'score': 0.9988006353378296,
+  'label': 'cat',
+  'box': {'xmin': 13, 'ymin': 52, 'xmax': 314, 'ymax': 470}},
+ {'score': 0.9986783862113953,
+  'label': 'cat',
+  'box': {'xmin': 345, 'ymin': 23, 'xmax': 640, 'ymax': 368}}]
+```
+
+Aquí obtenemos una lista de objetos detectados en la imagen, con un cuadro que rodea el objeto y una puntuación de confianza. Aquí está la imagen original a la derecha, con las predicciones mostradas a la izquierda:
```
Comment (osanseviero) [inline README_es.md:156]:
```suggestion
Aquí obtenemos una lista de objetos detectados en la imagen, con un cuadro que rodea el objeto y una puntuación de confianza. Aquí está la imagen original a la izquierda, con las predicciones mostradas a la derecha:
```

File: README_es.md [hunk 6]
```diff
@@ -0,0 +1,443 @@
...
+>>> tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+>>> model = TFAutoModel.from_pretrained("bert-base-uncased")
+
+>>> inputs = tokenizer("Hello world!", return_tensors="tf")
+>>> outputs = model(**inputs)
+```
+
+El tokenizador es responsable de todo el preprocesamiento que espera el modelo preentrenado y se puede llamar directamente en una sola cadena (como en los ejemplos anteriores) o en una lista. Dará como resultado un diccionario que puedes usar en el código descendente o simplemente pasarlo directamente a su modelo usando el operador de desempaquetado de argumento **.
+
+El modelo en si es un [Pytorch `nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) normal o un [TensorFlow `tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) (dependiendo De tu backend) que puedes usar de forma habitual. [Este tutorial](https://huggingface.co/docs/transformers/training) explica cómo integrar un modelo de este tipo en un ciclo de entrenamiento PyTorch o TensorFlow clásico, o como usar nuestra API `Trainer` para ajustar rápidamente un nuevo conjunto de datos.
```
Comment (osanseviero) [inline README_es.md:189]:
```suggestion
El modelo en si es un [Pytorch `nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) normal o un [TensorFlow `tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) (dependiendo de tu backend) que puedes usar de forma habitual. [Este tutorial](https://huggingface.co/docs/transformers/training) explica cómo integrar un modelo de este tipo en un ciclo de entrenamiento PyTorch o TensorFlow clásico, o como usar nuestra API `Trainer` para ajustar rápidamente un nuevo conjunto de datos.
```

File: README_es.md [hunk 7]
```diff
@@ -0,0 +1,443 @@
...
+    - Docenas de arquitecturas con más de 60 000 modelos preentrenados en todas las modalidades.
+
+1. Elija el marco adecuado para cada parte de la vida útil de un modelo:
+    - Entrene modelos de última generación en 3 líneas de código.
+    - Mueva un solo modelo entre los marcos TF2.0/PyTorch/JAX a voluntad.
+    - Elija sin problemas el marco adecuado para la formación, la evaluación y la producción.
+
+1. Personalice fácilmente un modelo o un ejemplo según sus necesidades:
+    - Proporcionamos ejemplos de cada arquitectura para reproducir los resultados publicados por sus autores originales..
+    - Los internos del modelo están expuestos lo más consistentemente posible..
```
Comment (osanseviero) [inline README_es.md:211]:
```suggestion
    - Las partes internas del modelo están expuestss lo más consistentemente posible.
```

**Review by osanseviero — APPROVED** [review]
Muchas gracias! 

I left some minor suggestions, but the translation is excellent! :fire: thanks a lot!

File: README_es.md (hunk 3 above)
Comment (Oussamaosman02) [reply README_es.md:99]: There you put `pregntas` instead off `preguntas`

File: README_es.md (hunk 7 above)
Comment (Oussamaosman02) [reply README_es.md:211]: There you wrote `expuestss` and should be `expuestas`

**Comment by Oussamaosman02** [comment]
@osanseviero  I have read and accept the suggestions you make, only few small mistakes that i commented.Thanks a lot for answering so fast!

File: README_es.md (hunk 3 above)
Comment (osanseviero) [reply README_es.md:99]: Fixed!

File: README_es.md (hunk 7 above)
Comment (osanseviero) [reply README_es.md:211]: Fixed!

**Comment by osanseviero** [comment]
Hey there! I fixed the 2 suggestions you mentioned :) feel free to commit them now :D

**Comment by sgugger** [comment]
Can you just run `make fix-copies` on your branch to fix the CLI issue? There is probably one or two models out of sync between the READMEs.

File: utils/check_copies.py [hunk 8]
```diff
@@ -70,6 +70,14 @@
             " {paper_authors}.{supplements}"
         ),
     },
+    "README_es.md": {
+        "start_prompt": "🤗 Transformers actualmente proporciona las siguientes arquitecturass",
```
Comment (sgugger) [inline utils/check_copies.py:74]: It looks like the util that checks for copies does not find this start prompt in the README in Spanish. Could you make sure it is correct?

**Comment by Oussamaosman02** [comment]
I hope now everything is right, sorry for the mistakes :smiling_face_with_tear:

**Comment by sgugger** [comment]
No worries, looking good now. Thanks again!
